.circleci/local_publish_helpers.sh

```diff
@@ -236,7 +236,7 @@ function retry {
 
     resetAwsAccountCredentials
     TEST_SUITE=${TEST_SUITE:-"TestSuiteNotSet"}
-    aws cloudwatch put-metric-data --metric-name FlakyE2ETests --namespace amplify-cli-e2e-tests --unit Count --value $n --dimensions testFile=$TEST_SUITE
+    aws cloudwatch put-metric-data --metric-name FlakyE2ETests --namespace amplify-cli-e2e-tests --unit Count --value $n --dimensions testFile=$TEST_SUITE || true
```

Comment:
why are we swallowing error ?

Reply:
Sometimes we reach a token expiration before this metric is collected; in those situations, we would like to ignore the error. Sometimes tests passes but token expires after test has completed, this causes unintentional "failures" which shouldn't be failures. A solution to token expiry is in progress

Reply:
Please leave comment in the code explaining this.